templates/Dockerfile-run

```diff
@@ -309,6 +309,33 @@ RUN \
         make install && \
         rm -rf ${DIR}
 
+## libxml2 - for libbluray
```

Comment:
the system package is too old or not consistent across variants ?

Reply:
it's inconsistent and therefore too old on some variants. I know that definitely when the images were using centos7 that was definitely too old to build a recent libbluray, and when I was testing the image builds I had a couple of failures that looked like they _might_ have been related to libxml2 being the wrong version so I figured that if there's a known hard dependency then might as well build a known version in the container to be copied over to the runtime.

The one image that I _know_ won't work if libxml2 isn't copied across is the nvidia runtime container, it just doesn't have libxml2 so it would need to be installed explicitly, which feels a little inconsistent, that was what caused the nvidia failures to happen in the previous commit, because the build time container for nvidia had libxml installed as a dependency of something else, but the runtime container was missing it, when searching for dependent libs for ffmpeg libxml that had been built fell to low in the dependency chain so didn't make it into the runtime container.